src/LAMMPS.jl

```diff
@@ -250,4 +269,11 @@ function gather_atoms(lmp::LMP, name, T, count)
     return data
 end
 
+function __init__()
+    if !MPI.Initialized()
```

Comment:
We should do this as the library for the user. The user might want to change the `threadlevel`

cc: @simonbyrne